### PR TITLE
OEUI-143: Roll out a beta release

### PR DESCRIPTION
**Pull request by larrystone**
## **JIRA TICKET NAME:**
[OEUI-143: Roll out a beta release](https://issues.openmrs.org/browse/OEUI-143)

### SUMMARY:
Setup dependencies for a beta release roll out of the Order Entry UI application for testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,3 +5,13 @@ after_success:
 - npm run coveralls
 install:
 - npm install
+before_deploy: npm run build
+deploy:
+  provider: bintray
+  file: "bintray.json"
+  user: "larrystone"
```
Comment (dkayiwa) [inline .travis.yml]: Have you looked at these?
https://github.com/openmrs/openmrs-owa-addonmanager/blob/master/.travis.yml
https://github.com/openmrs/openmrs-owa-cohortbuilder/blob/master/.travis.yml
https://github.com/openmrs/openmrs-owa-obsadmin/blob/master/.travis.yml

**Comment by larrystone** [comment]
Thanks for merging the PR. Please I need your help in adding my secret key to $BINTRAY_API_KEY on travis.
I will also need to create a git tag to version the release.

**Comment by dkayiwa** [comment]
Can you just use user rkorytkowski as for the other repos?

**Comment by larrystone** [comment]
Okay, will do that.

**Pull request merged**